docs/node-operators/get-started/setup-validator.md

```diff
@@ -314,13 +314,20 @@ To move on to self-staking, wait until you see your node on the validator list b
 
 Then, associate your tokens and nominate your node using the [governance dApp ↗](https://validator-testnet.governance.vega.xyz/staking/) or by interacting directly with the smart contract.
 
+## Forwarding Ethereum events
+Once your node is up and running, you'll need to maintain it, and ensure that it continues to take part in the network.
+
+Every time a method is called successfully on the smart contracts, for example a deposit on the collateral bridge, an event is emitted by the smart contract. Your validator node will be monitoring all blocks created by Ethereum and look for these events, and then forward them to all other validator nodes.
```

Comment:
First I'd suggest saying "forward them to the vega chain" rather than to "all other validator nodes", but more importantly a subtle point is that rather than actually forwarding the event to vega chain, they need to be *ready* to forward the event to the vega chain if selected.